### DISCUSSION TITLE
How can I create two different roles under the same company?

### DISCUSSION THREAD

**Discussion by karpathyo** [posquit0/Awesome-CV#455] [General]
I want to put two different roles under the last company section that I've worked, but tried different ways without success. Do you have any idea how to do it?

**Comment by OJFord** [comment]
#297 is the feature request for this, see there for some suggestions in the meantime.

**Reply by karpathyo** [reply]
> #297 is the feature request for this, see there for some suggestions in the meantime.

Thanks very much, I'll check it later.
